ee/desktop/user/server/server_darwin.go

```diff
@@ -6,17 +6,22 @@ package server
 import (
 	"crypto/ecdsa"
 	"fmt"
+	"log/slog"
 	"net/http"
 	"strings"
 
 	"github.com/kolide/krypto/pkg/echelper"
 	"github.com/kolide/krypto/pkg/secureenclave"
 )
 
-func (s *UserServer) createSecureEnclaveKey(w http.ResponseWriter, _ *http.Request) {
+func (s *UserServer) createSecureEnclaveKey(w http.ResponseWriter, r *http.Request) {
 	key, err := secureenclave.CreateKey()
 	if err != nil {
-		http.Error(w, fmt.Errorf("creating key: %w", err).Error(), http.StatusInternalServerError)
+		s.slogger.Log(r.Context(), slog.LevelDebug,
+			"secure enclave unavailable, could not create key",
+			"err", err,
+		)
+		http.Error(w, fmt.Errorf("secure enclave unavailable, could not creating key: %w", err).Error(), http.StatusServiceUnavailable)
```

Comment:
```suggestion
		http.Error(w, fmt.Errorf("secure enclave unavailable, could not create key: %w", err).Error(), http.StatusServiceUnavailable)
```

Reply:
I guess I don't know what these errors typically look like- just verifying, is that something we should expose directly in the response?

Reply:
they are just apple error code statuses like `errKCItemNotFound -25300` , there shouldn't be anything sensitive